.github/workflows/macos.yml

```diff
@@ -17,3 +17,19 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: ./scripts/benchmark.sh --transfer-count 4000
+
+  # Validation only. The linux / docs step validates and checks the diff.
+  # Also, this only validates that Docker on macOS works. It doesn't
+  # test that the commands run on macOS outside of Docker.
+  docs:
+    runs-on: ubuntu-latest
```

Comment:
@eatonphil should this be `macos-latest`?

Reply:
Oh, yes. It shouldn't make a huge difference since the bulk of the test runs in docker (unfortunately, and for the moment). But yes. Good catch.

Reply:
ah, looks like it's failing: https://github.com/tigerbeetledb/tigerbeetle/actions/runs/4292636474/jobs/7479321855

Reply:
Oof. I've disabled this macos test for now. I'll revisit it.